pocs/utils/images/__init__.py

```diff
@@ -92,44 +98,58 @@ def make_pretty_image(fname, timeout=15, **kwargs):  # pragma: no cover
         return _make_pretty_from_fits(fname, **kwargs)
 
 
-def _make_pretty_from_fits(fname=None, figsize=(10, 8), dpi=150, **kwargs):
+
+def _make_pretty_from_fits(fname=None, figsize=(10, 8), dpi=150, alpha=0.2, pad=3.0, **kwargs):
     header = getheader(fname)
     data = getdata(fname)
```

Comment:
`getdata` and `getheader` are mostly meant as convenience functions and I think aren't optimal if you need to get both of them anyway. Since we are blocking while making this image we should optimize a little. 

```
with fits.open(fname) as hdu:
    header = hdu[0].header
    data = hdu[0].data
```

Using the `with` context manager makes sure that the file is closed immediately after you get the header and data so doesn't keep the resources.